dogfood/main.tf

```diff
@@ -71,6 +71,50 @@ data "coder_parameter" "region" {
   # }
 }
 
+
```

Comment:
Can't you just push a new template and test it there? I believe that you don't need to sneak `security_groups` into the main template.

Reply:
Hi @mtojek I have rectified the issue and now the template is cleaner and only add a coder_app and required icon for gateway.